examples/models/rbc_dtcc_iid_2ar1.yaml

```diff
@@ -0,0 +1,92 @@
+name: Real Business Cycle
+
+model_type: dtcc
+
+symbols:
+
+   exogenous: [e_z, e_d]
+   states: [z, d, k]
+   controls: [n, i]
+   expectations: [m]
+   values: [V]
+   parameters: [beta, sigma, eta, chi, delta, alpha, rho, rho_d, zbar, dbar, sig_z, sig_d]
+   rewards: [u]
+
+definitions:
+    y: exp(z)*k^alpha*n^(1-alpha)
+    c: y - i
+    rk: alpha*y/k
+    w: (1-alpha)*y/n
+
+equations:
+
+    arbitrage:
+        - chi*n^eta*c^sigma - w                             | 0.0 <= n <= inf
+        - 1 - exp(d)*beta*(c/c(1))^(sigma)*(1-delta+rk(1))  | 0.0 <= i <= inf
+
+
+    transition:
+        - z = rho*z(-1) + e_z
+        - d = rho_d*d(-1) + e_d
+        - k = (1-delta)*k(-1) + i(-1)
+
+    value:
+        - V = c^(1-sigma)/(1-sigma) - chi*n^(1+eta)/(1+eta) + beta*V(1)
+
+    felicity:
+        - u =  c^(1-sigma)/(1-sigma) - chi*n^(1+eta)/(1+eta)
+
+    expectation:
+        - m = beta/c(1)^sigma*(1-delta+rk(1))
+
+    direct_response:
+        - n = ((1-alpha)*exp(z)*k^alpha*m/chi)^(1/(eta+alpha))
+        - i = exp(z)*k^alpha*n^(1-alpha) - (m)^(-1/sigma)
+
+calibration:
+
+    # parameters
+    beta : 0.99
+    phi: 1
+    delta : 0.025
+    alpha : 0.33
+    rho : 0.8
+    rho_d : 0.65
+    sigma: 5
+    eta: 1
+    sig_z: 0.016
+    sig_d: 0.025
+    zbar: 0
+    dbar: 0
+    chi : w/c^sigma/n^eta
+    c_i: 1.5
+    c_y: 0.5
+    e_z: 0.0
+    e_d: 0.0
+
+    # endogenous variables
+    n: 0.33
+    z: zbar
+    d: dbar
+    rk: 1/beta-1+delta
+    w: (1-alpha)*exp(z)*(k/n)^(alpha)
+    k: n/(rk/alpha)^(1/(1-alpha))
+    y: exp(z)*k^alpha*n^(1-alpha)
+    i: delta*k
+    c: y - i
+    V: log(c)/(1-beta)
+    u: c^(1-sigma)/(1-sigma) - chi*n^(1+eta)/(1+eta)
+    m: exp(d)*beta/c^sigma*(1-delta+rk)
+
+options:
+
+    exogenous: !Normal
+        sigma: [[sig_z**2, 0], [0, sig_d**2]]
```

Comment:
I can't remember if we handle the `**` in Julia or if we require `^`.

Does it work as is?